examples/resources/mondoo_integration_email/resource.tf

```diff
@@ -0,0 +1,26 @@
+provider "mondoo" {
+  space = "hungry-poet-123456"
+}
+
+# Setup the Email integration
+resource "mondoo_integration_email" "email_integration" {
+  name = "My Email Integration"
+
+  recipients = [
+    {
+      name          = "John Doe"
+      email         = "john@example.com"
+      is_default    = true
+      reference_url = "https://example.com"
+    },
+    {
+      name          = "Alice Doe"
+      email         = "alice@example.com"
+      is_default    = false
+      reference_url = "https://example.com"
+    }
+  ]
+
+  auto_create = true
```

Comment:
I think we want to set them to false as default.

Reply:
Die Parameter sind Optional, sollten also automatisch wenn nicht angegeben auf false gesetzt werden, da der default für ein Boolean in Golang false ist.

Oder meinst du wir sollten es im example als false setzen?

Reply:
Wenn du im example meinst könnten wir hier auch einfach ein Kommentar anfügen:
```terraform
provider "mondoo" {
  space = "hungry-poet-123456"
}
# Setup the Email integration
resource "mondoo_integration_email" "email_integration" {
  name = "My Email Integration"
  recipients = [
    {
      name          = "John Doe"
      email         = "john@example.com"
      is_default    = true # default is false
      reference_url = "https://example.com"
    },
    {
      name          = "Alice Doe"
      email         = "alice@example.com"
      reference_url = "https://example.com"
    }
  ]
  auto_create = true # default is false
  auto_close  = true # default is false
}
```

Reply:
oder aber auch im im Schema anmerken dass der default false ist